### PR TITLE
Fall clean-up

### PR DESCRIPTION
**Pull request by cloveras**
- Simplified (rewrote and reorganised) the callback sections
- Toned down "direct capture"
- Replaced some text with links to FAQ
- Proofreading, typos, formatting

Messy diff, so probably bets to see the final product: https://github.com/vippsas/vipps-ecom-api/blob/callback-cleanup/vipps-ecom-api.md

Slack: https://vippsas.slack.com/archives/C8HP1AJ79/p1665685862747249

### CODE REVIEW

File: vipps-ecom-api.md [hunk 1]
```diff
@@ -224,90 +221,86 @@ For example, if the vendor's name is "Acme AS" and the vendor offers two differe
...
 
-When you initiate a payment, it will only be *reserved* until you capture it.
-Vipps supports both *reserve capture* and *direct capture* payment flows.
+When you initiate a payment, it will normally only be *reserved* until you capture it.
 
-**Reserve capture** is the default. When you initiate a payment, it will be
-reserved until you capture it.
+This has some benefits:
+* If a payment has been _reserved_, the merchant can
+  make a `/cancel` call to immediately release the reservation and make available
```
Comment (flinkgutt) [inline vipps-ecom-api.md:244]: "make available in the customer's account." sounds clunky.

File: vipps-ecom-api.md [hunk 2]
```diff
@@ -761,64 +745,69 @@ public class UrlValidate {
...
-
-**Important:** Vipps callbacks require a response within 3 seconds; otherwise, they will time out.
-
-If you do not receive any confirmation on the payment request, you *must* call
-[`GET:/ecomm/v2/payments/{orderId}/details`](https://vippsas.github.io/vipps-developer-docs/api/ecom#tag/Vipps-eCom-API/operation/getPaymentDetailsUsingGET) to get the status of the payment.
-
-**Please note:** Do not rely only on callbacks alone, but also poll the `/details` endpoint
-as described in
+**Please note:** Callback offer a faster user experience than polling, but you
+cannot rely on callbacks alone. You must also poll
```
Comment (flinkgutt) [inline vipps-ecom-api.md:757]: Suggest we say `should not`.
`cannot rely on callbacks alone.` is a bit strong for my tastes.

File: vipps-ecom-api.md [hunk 3]
```diff
@@ -761,64 +745,69 @@ public class UrlValidate {
...
+  as it depends on network, firewalls and other factors that Vipps cannot control.
+  If the communication is broken during the process for some reason and Vipps
+  is not able to execute callback to the merchant's server, the callback will
+  not be retried.
+* Vipps callbacks require a response within 3 seconds; otherwise, they will time out.
+* Callback URLs must use HTTPS.
 
-See the FAQ:
-[Why do I not get callbacks from Vipps?](vipps-ecom-api-faq.md#why-do-i-not-get-callbacks-from-vipps)
+The callback request body is different for the different payment types:
```
Comment (flinkgutt) [inline vipps-ecom-api.md:771]: user-info flow?
Mention it perhaps

File: vipps-ecom-api.md [hunk 4]
```diff
@@ -1250,38 +1208,16 @@ there is a remaining reserved amount.
...
+It is also possible to explicitly release the remainin funds. See:
+[Cancelling a partially captured order](#cancelling-a-partially-captured-order).
 
 ## Cancel
 
 The Cancel request allows the merchant to cancel a reserved or initiated transaction.
 
-Please note that it is not possible to cancel a request that is over 6 months old.
+Please note that it is not possible to cancel a request that is over 180 days old.
+Attempting to calnce an older payment will result in a `HTTP 400 Bad Request`.
```
Comment (flinkgutt) [inline vipps-ecom-api.md:1220]: `calnce`, should be `cancel`.

**Review by flinkgutt — CHANGES_REQUESTED** [review]

File: vipps-ecom-api.md (hunk 3 above)
Comment (cloveras) [reply vipps-ecom-api.md:771]: Uh, then I need my own server to accept the callback.

Comment (cloveras) [reply vipps-ecom-api.md:771]: Hmm, the same as express, it seems.

File: vipps-ecom-api.md (hunk 1 above)
Comment (cloveras) [reply vipps-ecom-api.md:244]: Fixed, thanks!

**Pull request merged**